### DISCUSSION TITLE
PnP Search webpart not appearing when package deploy at site collection appcatalog level

### DISCUSSION THREAD

**Discussion by salil30july** [microsoft-search/pnp-modern-search#3519] [Q&A]
In installation instruction, it is mentioned that we can deploy the package site collection app catalog level. When we have tried deploying  at Site Collection AppCatalog level, none of the webpart was available even there was  no API Access level approval. When we deployed at tenant level, everything works fine.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/49209905/ac87f9f5-7d77-4c98-b2d9-2bbb4166356f)

**Comment by kasperbolarsen** [comment]
You definitely can deploy the PnP Modern Search package in the Site Collection app catalog. However, you will have to add the app, using the "gear" menu, before the web parts become available. 
Can you check and revert?
